Flow.Launcher/ViewModel/MainViewModel.cs

```diff
@@ -704,33 +704,30 @@ public async void ToggleFlowLauncher()
             }
             else
             {
-                switch (_settings.LastQueryMode)
-                {
-                    case LastQueryMode.Empty:
-                        ChangeQueryText(string.Empty);
-                        Application.Current.MainWindow.Opacity = 0; // Trick for no delay
-                        await Task.Delay(100);
-                        Application.Current.MainWindow.Opacity = 1;
-                        break;
-                    case LastQueryMode.Preserved:
-                        LastQuerySelected = true;
-                        break;
-                    case LastQueryMode.Selected:
-                        LastQuerySelected = false;
-                        break;
-                    default:
-                        throw new ArgumentException($"wrong LastQueryMode: <{_settings.LastQueryMode}>");
-                }
-                MainWindowVisibility = Visibility.Collapsed;
+                Hide();
             }
         }
 
-        public void Hide()
+        public async void Hide()
```

Comment:
why add async here? @onesounds / @taooceros

Reply:
for handling the task delay for fixing empty query issue.